### PR TITLE
Generate collections from schema section

### PR DESCRIPTION
**Pull request by ArifPMan**
Adding a new section to API chapter

### CODE REVIEW

File: v6/postman/design_and_develop_apis/the_api_workflow.md [hunk 1]
```diff
@@ -32,7 +32,17 @@ The schema editor provides you options to beautify the content, wrap text, copy
 
 [![api schema add](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Schema-Editor2.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Schema-Editor2.png)
 
-Click **Generate Collection** to generate a collection from this schema. The collection will not automatically link to an API. You will need to link the collection explicitly to an API or a specific API version using a version tag.  
+Click **Generate Collection** to generate a collection from this schema. The collection will not automatically link to an API. You will need to link the collection explicitly to an API or a specific API version using a version tag.
+
+##### Generating collection from a schema
```
Comment (DannyDainton) [inline v6/postman/design_and_develop_apis/the_api_workflow.md]: Should this be either 'Generate a collection from a schema' or 'Generating a collection from a schema' or something different - Not sure?

File: v6/postman/design_and_develop_apis/the_api_workflow.md [hunk 2]
```diff
@@ -32,7 +32,17 @@ The schema editor provides you options to beautify the content, wrap text, copy
...
+
+This is a powerful feature that will let you create collections from a schema as and when you want emphasizing the API-first development approach.
```
Comment (DannyDainton) [inline v6/postman/design_and_develop_apis/the_api_workflow.md]: Feels like this needs some grammar changes. This doesn't sound quite right - _as and when you want emphasizing the API-first development approach._

File: v6/postman/design_and_develop_apis/the_api_workflow.md [hunk 3]
```diff
@@ -32,7 +32,17 @@ The schema editor provides you options to beautify the content, wrap text, copy
...
+
+Clicking **Generate Collections** button generates a collection which adheres to your schema and puts it in your workspace sidebar. Now, you can do whatever you wish to - you can add documentation to it, you can attach test suites, and you can even use it to generate a quick collection for collaboration on some thing without affecting the source of truth which is an API (by spinning off a collection from the schema and sharing it).
```
Comment (DannyDainton) [inline v6/postman/design_and_develop_apis/the_api_workflow.md]: - This should be "Clicking the **Generate Collection** button"
- Is this missing _it_ on the end? "Now, you can do whatever you wish to"

File: v6/postman/design_and_develop_apis/the_api_workflow.md [hunk 4]
```diff
@@ -32,7 +32,17 @@ The schema editor provides you options to beautify the content, wrap text, copy
...
+
+##### Generating collection from a schema
+
+This is a powerful feature that will let you create collections from a schema as and when you want emphasizing the API-first development approach.
+
+Clicking **Generate Collections** button generates a collection which adheres to your schema and puts it in your workspace sidebar. Now, you can do whatever you wish to - you can add documentation to it, you can attach test suites, and you can even use it to generate a quick collection for collaboration on some thing without affecting the source of truth which is an API (by spinning off a collection from the schema and sharing it). 
+
+What this essentially means you don't need to import your schema file again and again to generate collections. People who have repetitive workflows can import their schemas in Postman and can generate collections on the run. You should re-import the schema only when the schema changes. This would pave way for the API-first development approach.
```
Comment (DannyDainton) [inline v6/postman/design_and_develop_apis/the_api_workflow.md]: Not really sure what this means or is telling me? Could be re-worded?

Also, should it be _pave the way_

File: v6/postman/design_and_develop_apis/the_api_workflow.md [hunk 5]
```diff
@@ -32,7 +32,17 @@ The schema editor provides you options to beautify the content, wrap text, copy
...
+
+##### Generating collection from a schema
+
+This is a powerful feature that will let you create collections from a schema as and when you want emphasizing the API-first development approach.
+
+Clicking **Generate Collections** button generates a collection which adheres to your schema and puts it in your workspace sidebar. Now, you can do whatever you wish to - you can add documentation to it, you can attach test suites, and you can even use it to generate a quick collection for collaboration on some thing without affecting the source of truth which is an API (by spinning off a collection from the schema and sharing it). 
+
+What this essentially means you don't need to import your schema file again and again to generate collections. People who have repetitive workflows can import their schemas in Postman and can generate collections on the run. You should re-import the schema only when the schema changes. This would pave way for the API-first development approach.   
+
+You can quickly generate a collection from a schema by clicking **Generate Collection** button.
```
Comment (DannyDainton) [inline v6/postman/design_and_develop_apis/the_api_workflow.md]: _clicking the **Generate Collection** button_ ?

File: v6/postman/design_and_develop_apis/the_api_workflow.md (hunk 1 above)
Comment (kaustavdm) [reply v6/postman/design_and_develop_apis/the_api_workflow.md]: Should be `Generate a collection from a schema`. Also, this level of nesting of heading looks too deep. I see @tristandenyer reordering heading hierarchy. We should follow a common convention.

File: v6/postman/design_and_develop_apis/the_api_workflow.md (hunk 2 above)
Comment (kaustavdm) [reply v6/postman/design_and_develop_apis/the_api_workflow.md]: Agree with @DannyDainton about grammar changes. Plus, this feature is actually not that powerful yet till we have syncing between schema and generated collection. Currently this is a one time approach, and each time you make changes to the schema, you will have to generate a new collection. So, possibly de-emphasize this statement.

File: v6/postman/design_and_develop_apis/the_api_workflow.md (hunk 4 above)
Comment (kaustavdm) [reply v6/postman/design_and_develop_apis/the_api_workflow.md]: Just shorten it to something like, `It means...`. Also, let's not overuse the API-first development approach. Sounds repetitive at this stage.

**Comment by ArifPMan** [comment]
@tristandenyer @DannyDainton @kaustavdm - This is still work under progress, I have just randomly put down my thoughts. Why are people reviewing this on the fly? The first draft of this isn't ready yet. 

@tristandenyer - Can you please ensure me how I should author source files without providing access to folks until the first draft is ready? I guess we discussed this earlier too. Would like to have a meeting with you tomorrow.

**Comment by a85** [comment]
@ArifPMan We can have a meeting and everything but I'd like to flag that the current status of docs is not fine with me. PRs should not be open endlessly. Docs should be 100% complete and Tristan shouldn't have to clean up branches in docs. This was open since May 25 which is crazy. I can set up a DevRel sync to get everybody on the same page.

**Comment by ArifPMan** [comment]
Folks can review this PR and provide their f/b comments.

File: v6/postman/design_and_develop_apis/the_api_workflow.md (hunk 1 above)
Comment (tristandenyer) [reply v6/postman/design_and_develop_apis/the_api_workflow.md]: Common convention for headings should be to start with an H2 (##), and go in from there, as needed. The title of the page (which is at the very top of the .md file) is printed as an H1. Many of our docs start with an H3 or H4, which is not semantically correct.

**Pull request merged**